rest/admin_api_test.go

```diff
@@ -3483,5 +3483,17 @@ func TestDbConfigDoesNotIncludeCredentials(t *testing.T) {
 	assert.Equal(t, "", dbConfig.CACertPath)
 	assert.Equal(t, "", dbConfig.CertPath)
 	assert.Equal(t, "", dbConfig.KeyPath)
+}
+
+func TestInvalidDBConfig(t *testing.T) {
+	rt := NewRestTester(t, nil)
+	defer rt.Close()
+
+	resp := rt.SendAdminRequest("PUT", "/db/_config", "{}")
+	assertStatus(t, resp, http.StatusCreated)
+
+	resp = rt.SendAdminRequest("PUT", "/db/_config", `{"sync": "function(){"}`)
+	assertStatus(t, resp, http.StatusBadRequest)
```

Comment:
Can we test the `/_config/sync` and `/_config/import_filter` endpoints too (and add the appropriate validation that they don't have)

Reply:
Sorry... Accidentally went into autopilot and managed to squash my commits... 
Thats now done though